src/classic/theme.js

```diff
@@ -178,7 +178,33 @@ function getTheme({ style, name }) {
       "panelInput.border": pick({ light: primer.gray[2], dark: primer.gray[1] }),
 
       "terminal.foreground": primer.gray[6],
+      "terminal.background": primer.gray[0],
```

Comment:
I don't think we need to set the terminal background color. But the rest of your changes LGTM 👍

Reply:
thx! Removed background color setting!